comments-bundle/src/Resources/contao/classes/Comments.php

```diff
@@ -247,7 +247,6 @@ protected function renderCommentForm(FrontendTemplate $objTemplate, \stdClass $o
 		// Initialize the widgets
 		foreach ($arrFields as $arrField)
 		{
-			/** @var Widget $strClass */
```

Comment:
How can the class be inferred here? Shouldn't that be `/** @var Widget|null $strClass */`?

Reply:
No, `$strClass` is a string, isn't it?

Reply:
You're right, I looked at the wrong line :see_no_evil:.